apps/web/src/components/v2Editor/customBlocks/python/PythonOutput.tsx

```diff
@@ -106,16 +107,33 @@ interface ItemProps {
   canFixWithAI: boolean
 }
 export function PythonOutput(props: ItemProps) {
+  const onExportToPNG = () => {
+    if (props.output.type !== 'image' || props.output.format !== 'png') return
+
+    downloadFile(
+      `data:image/${props.output.format};base64, ${props.output.data}`,
+      'Generated Python image'
+    )
+  }
+
   switch (props.output.type) {
     case 'image':
       switch (props.output.format) {
         case 'png':
           return (
-            <img
-              className="printable-block"
-              alt="generated image"
-              src={`data:image/${props.output.format};base64, ${props.output.data}`}
-            />
+            <>
+              <img
+                className="printable-block"
+                alt="generated image"
+                src={`data:image/${props.output.format};base64, ${props.output.data}`}
+              />
+              <button
+                className="absolute bottom-0 bg-white rounded-md border border-gray-200 p-1 hover:bg-gray-50 z-10 right-0 text-xs text-gray-400"
```

Comment:
In my local tests this button's borders look slightly weird because it's covering the block's border (and all sides are rounded).

```suggestion
                className="absolute bottom-[1px] right-[1px] bottom-0 bg-white rounded-tl-md rounded-br-md border border-gray-200 p-1 hover:bg-gray-50 z-10 right-0 text-xs text-gray-400"
```

This fixes it (I moved it a px in each axis + fixed the border radius)

<img width="68" alt="Screenshot 2024-09-15 at 16 26 25" src="https://github.com/user-attachments/assets/0292e896-3975-4417-9c49-6e59e04a00fe">

Reply:
I made the button float because of this weird Windows' scrollbar. And now I think it makes more sense due to the multi-output thing.

Here is a preview of it:

![image](https://github.com/user-attachments/assets/260cf431-196d-4d45-b21e-0448d9eea891)

Reply:
@vtfg thanks for this! The issue here is that it will look weird on Mac OS, which doesn't show the scrollbar (and most of our users are on Macs too)

 I think that the 1px float and the rounded borders are fine and what we should actually do is handle the scrollbars on Windows separately. In fact, they should not appear on Windows too (we have done this elsewhere).
 
 What do you think of adding these CSS changes and then I'd love it if you want to tackle updating the scrollbars so they don't appear on Windows (I use a mac, so I have to go through a lot of work to test on Windows, if you have one that'd be easier).

Reply:
@lucasfcosta I've commited the suggested CSS changes. 

I don't know if they'll have the expected impact because of the padding applied to the collapsable component, making these buttons not too close to the container's border (at least on Windows).

![image](https://github.com/user-attachments/assets/bb335df6-4793-4a55-8e18-96c6a8ceb265)